packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
+
```

Comment:
nitpicky, but this newline can be removed. The fewer line changes the better

Reply:
Done :D